### PR TITLE
Saas-1304 DBaaS Landing Page 

### PR DESCRIPTION
**Pull request by doracretu3pillar**

### CODE REVIEW

File: src/components/Root/Main.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ import {
   K8sClusterCreationPage,
```
Comment (nicolalamacchia) [inline src/components/Root/Main.tsx:11]: You should remove `K8sClusterCreationPage` if you don't want to reuse it

File: src/core/routes.ts [hunk 2]
```diff
@@ -17,6 +17,7 @@ export const Routes = {
   welcome: '/welcome',
   dashboard: '/dashboard',
   kubernetes: '/kubernetes',
```
Comment (nicolalamacchia) [inline src/core/routes.ts:19]: we no longer need this if we're using a different URL

File: src/core/routes.ts [hunk 3]
```diff
@@ -25,4 +26,5 @@ export const RouteNames = {
   [Routes.instances]: 'PMM Instances',
   [Routes.kubernetes]: 'Free Kubernetes Cluster',
```
Comment (nicolalamacchia) [inline src/core/routes.ts:27]: Remove this as well

**Review by nicolalamacchia — APPROVED** [review]

File: src/components/Root/Main.tsx (hunk 1 above)
Comment (doracretu3pillar) [reply src/components/Root/Main.tsx]: removed

File: src/core/routes.ts (hunk 3 above)
Comment (doracretu3pillar) [reply src/core/routes.ts]: removed

File: src/core/routes.ts (hunk 2 above)
Comment (doracretu3pillar) [reply src/core/routes.ts]: removed

**Pull request merged**